src/SymptomChecker/CheckIn.tsx

```diff
@@ -121,13 +138,44 @@ const FeelingGoodContent = () => {
   )
 }
 
-const FeelingNotWellContent = () => {
+interface FeelingNotWellContentProps {
+  symptoms: string[]
+  healthAssessment: HealthAssessment
+}
+
+const FeelingNotWellContent: FunctionComponent<FeelingNotWellContentProps> = ({
+  symptoms,
+  healthAssessment,
+}) => {
   const { t } = useTranslation()
 
+  const determineHealthAssessmentText = () => {
+    switch (healthAssessment) {
+      case HealthAssessment.GetTested:
+        return t("symptom_checker.get_tested")
+      case HealthAssessment.FollowHAGuidance:
```

Comment:
```suggestion
      case HealthAssessment.NotAtRisk:
```

This naming will leave us open to keep the notion of the assessment flow separate from the recommendation flow and allow them to grow/change independently. for example we might have a future world where we want something like

```typescript
type HealthAssessment = NotStarted | InProgress | AtRisk | NotAtRisk

type HealthRecommendation = GetTested | CallDoctor | DoNothing
```

its a bit premature to finalize what exactly these states should be but we probably want to be moving towards something like this to support the self assessment feature.

Reply:
Sounds good!